Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -394,6 +394,10 @@ void SeedFinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
         if (m_config.arithmeticAverageCotTheta) {
           // use arithmetic average
           cotThetaAvg2 = std::pow((cotThetaB + cotThetaT) / 2, 2);
+        } else {
+          if (cotThetaAvg2 < 0) {
+            continue;
+          }
         }
```

Comment:
```suggestion
        } else if (cotThetaAvg2 < 0) {
          continue;
        }
```

Reply:
?

Reply:
to me it would be more readable to have the `cotThetaAvg2 = cotThetaB * cotThetaT;` in the `else` and leave it uninitialized up there.

Reply:
Does that even work without a warning?

Reply:
Just readability (sry, should have noted that)

I think we can't leave it uninitialized, since we use it later ...

Reply:
Let's see.

Reply:
yeah not sure how hard clang-tidy will pick on us

Reply:
Ok, same result as before, small output changes.